lib/rules/generator-star-spacing.js

```diff
@@ -29,7 +29,67 @@ module.exports = {
                         type: "object",
                         properties: {
                             before: { type: "boolean" },
-                            after: { type: "boolean" }
+                            after: { type: "boolean" },
+                            named: {
+                                oneOf: [
+                                    {
+                                        enum: ["before", "after", "both", "neither"]
+                                    },
+                                    {
+                                        type: "object",
+                                        properties: {
+                                            before: { type: "boolean" },
+                                            after: { type: "boolean" }
+                                        },
+                                        additionalProperties: false
+                                    }
```

Comment:
Nitpick: This schema is repeated several times. Can you extract the object into a constant?

```js
const SPECIFIC_CONFIG_SCHEMA = {
    oneOf: [
        // ...
    ]
};
```